template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CmTemplateProcessor.java

```diff
@@ -961,4 +964,30 @@ public Optional<ApiClusterTemplateConfig> getRoleConfig(String serviceType, Stri
                         .filter(config -> configName.equals(config.getName()))
                         .findAny());
     }
+
+    public List<SafetyValve> getSafetyValves() {
+        List<SafetyValve> safetyValves = new ArrayList<>();
+        List<ApiClusterTemplateConfig> apiClusterTemplateConfigList;
+        for (ApiClusterTemplateService apiClusterTemplateService : cmTemplate.getServices()) {
+            apiClusterTemplateConfigList = (!CollectionUtils.isEmpty(apiClusterTemplateService.getServiceConfigs()) ?
+                    apiClusterTemplateService.getServiceConfigs() : Collections.emptyList());
```

Comment:
Nit: The unnecessary outer parentheses will be flagged as errors later when migrating to CheckStyle 10.6.0. See the many related changes we had to make in #14671.

Reply:
Done